sm/engine/dataset_manager.py

```diff
@@ -252,11 +254,23 @@ def _add_zoom_optical_images(self, ds, img_id, transform, zoom_levels):
         self._db.alter(DEL_OPTICAL_IMAGE, params=(ds.id,))
         self._db.insert(INS_OPTICAL_IMAGE, rows=rows)
 
+    def _add_thumbnail_optical_image(self, ds, img_id, transform):
+        size = 200, 200
+        self._db.alter(UPD_DATASET_THUMB_OPTICAL_IMAGE, params=(None, ds.id,))
+        dims = self._annotation_image_shape(ds)
+        optical_img = self._img_store.get_image_by_id('fs', 'raw_optical_image', img_id)
+        img = optical_img = self._transform_scan(optical_img, transform, dims, zoom=1)
```

Comment:
It's unnecessary that the same value is assigned to 2 different variables in this line. Please remove `optical_img = ` from this line and change the below `_save_jpeg` line to use `img` instead of `optical_img`.

Reply:
Sorry, I didn't notice that when was changing this line. 
thanks, @LachlanStuart